src/internal/constants/supportedProperties.ts

```diff
@@ -3,9 +3,21 @@ import { AXIS } from "./axis";
 
 const BASE_PROPERTIES = ["dataLabels.visible", "grid", "legend", "colorMapping"];
 
-const BASE_X_AXIS_PROPERTIES = ["xaxis.labelsEnabled", "xaxis.rotation", "xaxis.visible"];
+const BASE_X_AXIS_PROPERTIES = [
+    "xaxis.labelsEnabled",
+    "xaxis.rotation",
+    "xaxis.visible",
+    "xaxis.name.visible",
+    "xaxis.name.position",
+];
 
-const BASE_Y_AXIS_PROPERTIES = ["yaxis.rotation", "yaxis.labelsEnabled", "yaxis.visible"];
+const BASE_Y_AXIS_PROPERTIES = [
+    "yaxis.rotation",
+    "yaxis.labelsEnabled",
```

Comment:
should use the same order with `BASE_X_AXIS_PROPERTIES`
first `labelsEnabled`
second `rotation `

Reply:
Fixed